pygmt/src/meca.py

```diff
@@ -79,10 +79,13 @@ def data_format_code(convention, component="full"):
     B="frame",
     C="cmap",
     E="extensionfill",
+    Fr="box",
```

Comment:
I'm unsure if `box` is a good name here, because `box` is usually used as an alias of `-F` option for `legend`, `inset`, `colorbar`, `basemap` and others.

Maybe we can rename it to `labelbox`?

Reply:
I am fine with `labelbox` for **-Fr**.
I was actually a bit wondering, whether people might think that this parameter adds a box behind the beachball, so `labelbox` is also more descriptive.